feature-configs/demo/ptp/ptpconfig-grandmaster.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: ptp.openshift.io/v1
```

Comment:
Would it make sense to provide a default for these manifests and only the kustomizations under demo?

Reply:
this is not a production way you should not configure the master to be one of the nodes.
All the nodes in the cluster should only be slaves, we do it like that in our environment for testing (we don't have an external device to sync from) this is why this file is in the demo folder

Reply:
Ah, got it. Makes sense, thanks for the explanation